event-sourcing/event-repository/event-repository-jdbc/src/main/java/uk/gov/justice/services/eventsourcing/repository/jdbc/EventStreamRepository.java

```diff
@@ -0,0 +1,17 @@
+package uk.gov.justice.services.eventsourcing.repository.jdbc;
+
+import uk.gov.justice.services.eventsourcing.repository.jdbc.eventstream.EventStreamObject;
+
+import java.util.UUID;
+import java.util.stream.Stream;
+
+public interface EventStreamRepository {
+
+    Stream<EventStreamObject> getBySequenceId(final Long sequenceNumber);
+
+    void markActive(final UUID streamId, final boolean active);
```

Comment:
again, is it consistent to use streamId only in one place?

Do we need to retrieve the EventStream by StreamId?

Reply:
No requirement right now retrieve by Stream Id also changed getBySequenceId to getEventStreamsFromSequenceNumber